src/commands/lambda/__tests__/flare.test.ts

```diff
@@ -50,7 +53,8 @@ const MOCK_FOLDER_PATH = path.join(MOCK_CWD, MOCK_FOLDER_NAME)
 const MOCK_FILE_NAME = 'function_config.json'
 const MOCK_FILES = new Set([MOCK_FILE_NAME, 'file1.csv', 'file2.csv', 'file3.csv'])
 const MOCK_ZIP_PATH = 'output.zip'
-const MOCK_REQUIRED_FLAGS = ['lambda', 'flare', '-f', 'func', '-r', 'us-west-2', '-c', '123', '-e', 'test@test.com']
+const MOCK_REGION = 'us-east-1'
```

Comment:
Looks good overall, the only thing I didn't quite get was the updating in every command as `MOCK_REGION`, I understand in the future we would allow the user to flare multiple functions, which is going to make testing harder if we keep this approach.

Reply:
Ah good point. In the future I won't use `MOCK_REGION` if the function specified is a function ARN.
This will only be used in the cases where `--region` is required as a flag